application/shoot_and_kill.py

```diff
@@ -0,0 +1,31 @@
+from application.action import Action
+from constans.constants_scores import KILL
+from application.utils import target_position_within_bounds
+from exceptions.personal_exceptions import invalidMoveException
+from game.board import Board
+from game.cell import Cell
+from game.character import Character
+from game.player import Player
```

Comment:
NIT:
It's a good practice to order the imports alphabetically 😄

Reply:
I'm going to modify it, thanks!